src/Uno.UWP/Storage/Pickers/FolderPicker.skia.cs

```diff
@@ -0,0 +1,27 @@
+﻿#nullable enable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.Extensions.Storage.Pickers;
+using Uno.Foundation.Extensibility;
+
+namespace Windows.Storage.Pickers
+{
+	public partial class FolderPicker
+    {
+		private IFolderPickerExtension? _folderPickerExtension;
+
+		partial void InitializePlatform() => ApiExtensibility.CreateInstance(this, out _folderPickerExtension);
+
+		private async Task<StorageFolder?> PickSingleFolderTaskAsync(CancellationToken token)
+		{
+			if (_folderPickerExtension == null)
+			{
+				throw new NotSupportedException("FileOpenPicker is not supported on this target.");
```

Comment:
Maybe the name "FileOpenPicker" is not correct here ;-)

Reply:
Fixd :-)